src/handlers/CloseBtn.ts

```
@@ -6,6 +6,7 @@ import { SuccessStatusComponent } from "../utils/embeds"
 import { CloseId } from "../utils/interactionIds"
 import { rerenderDashboard } from "../utils/misc"
 import { CheckRights } from "../utils/preconditions"
+import { getT } from "../generated/i18n"
```

ESLint

[error] Unable to resolve path to module '../generated/i18n'.

(import-x/no-unresolved)